shopinvader_partner_firstname/tests/test_firstname.py

```diff
@@ -1,25 +1,57 @@
 # Copyright 2017 Akretion (http://www.akretion.com).
```

Comment:
I think this old copyright is wrong as I probably copy/pasted the header: I've created the module in 2019. 
Please set the year to 2019 and remove Akretion.